### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by skuusela**
Signed-off-by: Simo Kuusela simo.kuusela@intel.com

### CODE REVIEW

**Comment by igor-stoppa** [comment]
@skuusela could you please also add a note about the delay that had to be increased?
that doesn't seem right and I'd like to have some reminder about it

**Comment by skuusela** [comment]
@igor-stoppa Done.

File: packaging/pem.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+* 2016-07-27 Simo Kuusela <simo.kuusela@intel.com> 0.1.11
+  - Make PEM compatible with both Python 2 and Python 3.
+  - In src/emulator.py/_transmit_with_retry, timeout value
+    for _phys.receive_message had to be increased, because
+    with Python 3 it didn't work that well with 200ms.
+
```
Comment (igor-stoppa) [inline packaging/pem.changes:6]: @skuusela apologies, I was not sufficiently clear, this looks good, but could you please add also a comment in the code? The release notes is not exactly the first place where one would look.

Comment (skuusela) [inline packaging/pem.changes:6]: @igor-stoppa Added comment in the code as well.

**Comment by igor-stoppa** [comment]
LGTM

**Pull request merged**